Dockerfile

```diff
@@ -1,7 +1,20 @@
-FROM scratch
-COPY ./cacert.pem /etc/ssl/certs/ca-certificates.crt
-COPY cardigann-linux-amd64 /cardigann
+FROM alpine:latest
+
+ENV CARDIGANN_PKG cardigann-stable-linux-amd64.tgz
+ENV CONFIG_DIR /etc/cardigann
+
+ADD https://bin.equinox.io/c/3u8U4iwUn6o/${CARDIGANN_PKG} /
```

Comment:
The issue with this rather than adding it from the local filesystem is that it's out of date with the build. The current process builds a docker image for each changeset, where as this would build on with the last release.